front/src/applications/operationalStudies/views/Scenario.jsx

```diff
@@ -69,7 +70,7 @@ export default function Scenario() {
       setScenario(result);
       dispatch(updateTimetableID(result.timetable));
       dispatch(updateInfraID(result.infra));
-      getTimetable(result.timetable);
+      getTimetable(timetableID, result.timetable);
```

Comment:
```suggestion
     await getTimetable(result.timetable ? result.timetable : timetableID);
```
Should give only 1 argument